static/css/restyle.css

```diff
@@ -0,0 +1,695 @@
+/* Hide outlines Firefox shows on every click */
+* {
+  outline: none;
+}
+
+/* */
+#tabzilla:before {
+  background: none;
+}
+
+/* */
+body {
+  font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
+  font-style: normal;
+  font-size: 0.875rem;
+  line-height: 1.5;
+  color: #333;
+  background: #FFF none repeat scroll 0% 0%;
+  min-width: 1018px;
+}
+
+h1 {
+  font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
+  font-style: normal;
+  font-size: 2.25rem;
+}
+
+/* Main Content Wraper */
+#background-wrapper {
+  border-top: 2px solid #FFF;
+  background-color: #0C99D5;
+  width: 100%;
+  position: absolute;
+  height: 420px;
+}
+/* Main Content Wraper Background - General
+ blue-pattern, blue-gradient, white overlay
+ (sharp-cut at 130px) */
+#background-wrapper {
+  background-image:
+    linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, ) 0px,
+    rgba(255, 255, 255, 0) 100px
+  ),
+  linear-gradient(
+    to bottom,
+    rgba( 12, 153, 213, 0) 0px,
+    rgba( 12, 153, 213, 1) 131px,
+    rgba(255, 255, 255, 1) 131px
+  ),
+  repeating-linear-gradient(
+    118deg,
+    rgba(12, 153, 213, 0) 0px,
+    #00A7E0 550px,
+    rgba(12, 153, 213, 0) 550px,
+    #00A7E0 800px,
+    rgba(12, 153,213, 0) 800px,
+    #00A7E0 950px
+  );
+  background-position:
+    0px 130px,
+    0px 0px,
+    0px 0px;
+}
+/* Main Content Wraper Background  - Home and Add-on Details page
+blue-pattern, blue-gradient, white overlay
+(sharp-cut at 416px) */
+.home #background-wrapper,
+.addon-details #background-wrapper {
+  background-image:
+    linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, 0) 0px,
+    rgba(255, 255, 255, 0) 100px
+  ),
+  linear-gradient(
+    to bottom,
+    rgba( 12, 153, 213, 0) 0px,
+    rgba( 12, 153, 213, 1) 417px,
+    rgba(255, 255, 255, 1) 417px
+  ),
+  repeating-linear-gradient(
+    118deg,
+    rgba(12, 153, 213, 0) 0px,
+    #00A7E0 550px,
+    rgba(12, 153, 213, 0) 550px,
+    #00A7E0 800px,
+    rgba(12, 153,213, 0) 800px,
+    #00A7E0 950px
+  );
+  background-position:
+    0px 416px,
+    0px 0px,
+    0px 0px;
+}
+/* hide overflow to cover streching of bg in
+ .addon-details .primary:first-of-type:before */
+.addon-details #background-wrapper {
+  overflow: hidden;
+}
+
+
+.amo-header {
+  font-family: inherit;
+}
+.menu-nav {
+  font-size: 0.75rem;
+  color: #fff;
+}
+.menu-nav a,
+.amo-header ul a {
+  color: #fff;
+}
+#masthead h1 {
+  font: inherit;
+  font-size: 3.5rem;
+  line-height: 3rem;
+  letter-spacing: -1px;
+  text-transform: none;
+}
+.site-title a {
+  color: #fff;
+  text-decoration: none;
+}
+.site-title a:focus, .site-title a:hover {
+  color: #fff;
+  text-shadow: none;
+}
+.site-title a img {
+  visibility: hidden;
+  width: 70px;
+}
+.site-title a:before {
+  content: "";
+  background-image: url("https://mozorg.cdn.mozilla.net/media/img/firefox/family/nav-sprite7-high-res.25f40fe41fe2.png");
```

Comment:
Would it make sense to not hardwire these in case further iteration is required? It will make it easier to make changes in local dev.